### PR TITLE
Reverting the graphsdk to azcli

### PR DESCRIPTION
**Pull request by Vidya2606**
# Description

I'm reverting from graphsdk to azcli because few of the functions for which I made the changes would require registration to Graph App which means the user had to do a device code login in the web browser which seemed to be a bad user experience, so we decided to keep the functions as is.

getAppObjectId: https://github.com/Azure/draft/pull/260 
assignSpRole: https://github.com/Azure/draft/pull/272

Fixes # (issue)
Feature # (details)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Is it a  breaking change which will impact consuming tool(s).

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/setup-gh.go [hunk 1]
```diff
@@ -44,20 +39,6 @@ application and service principle, and will configure that application to trust
 
 			sc.AzClient.AzTenantClient = client
 
-			graphClient, err := createGraphClient(azCred)
```
Comment (Tatsinnit) [inline cmd/setup-gh.go:47]: 💡 Thanks for the ping, if there is any previous work which this work revert can you please att the commit hash for that here or in description of this PR, please :) thank you! will make it easy to quickly review. 

Thanks heaps again.

Comment (Vidya2606) [reply cmd/setup-gh.go:47]: I have added the reference PR's in the description above :)

File: pkg/providers/azure.go [hunk 2]
```diff
@@ -320,15 +312,22 @@ func (sc *SetUpCmd) createFederatedCredentials() error {
 
 }
 
-func (sc *SetUpCmd) getAppObjectId(ctx context.Context) error {
+func (sc *SetUpCmd) getAppObjectId() error {
 	log.Debug("Fetching Azure application object ID")
 
-	appID, err := sc.AzClient.GraphClient.GetApplicationObjectId(ctx, sc.appId)
+	getObjectIdCmd := exec.Command("az", "ad", "app", "show", "--only-show-errors", "--id", sc.appId, "--query", "id")
```
Comment (Tatsinnit) [inline pkg/providers/azure.go:318]: ☕️ thanks for this, I am just curious why this is getting reverted? Using sdk for anything is better than using exec.command?

**Review by Tatsinnit — COMMENTED** [review]
☕️ thanks heaps Vidya!! since it’s just revert seems ok to me at revert level 😊 but I have asked a question and might repeat here for archive, it will be worth to add context for this revert as to why we are going back to exec.command?

Thank you so much as always for kind ping!! I will lil bird fyi @davidgamero for wiz look and eyes.

File: pkg/providers/azure.go (hunk 2 above)
Comment (Vidya2606) [reply pkg/providers/azure.go:318]: sorry for the confusion, I have updated in the description let me know if you have any other questions.

**Review by bfoley13 — APPROVED** [review]
This looks good for me, @davidgamero for final approval

**Review by davidgamero — APPROVED** [review]
yes looks good to me, validated manually with a full deployment to an aks cluster.

**Pull request merged**